tests/bootstrap.php

```diff
@@ -21,6 +21,9 @@
 require $test_root . '/includes/functions.php';
 
 function _manually_load_plugin() {
+	// Make sure the plugin is "enabled"
+	define( 'S3_UPLOADS_AUTOENABLE', true );
```

Comment:
That's perfectly fine, and I can remove this. I was setting the constant to `true` in attempts to get Travis to behave properly when running CI tests (see #31). The tests on Travis are apparently running without _any_ of the S3 constants defined when I push the patch, which causes the tests to fail entirely as the inclusion of various classes never fires.